corehq/apps/app_manager/views.py

```diff
@@ -1266,13 +1270,14 @@ def default_new_app(request, domain):
         domain, _("Untitled Application"), lang=lang,
         application_version=APP_V2
     )
-    app.add_module(Module.new_module(_("Untitled Module"), lang))
-    app.new_form(0, "Untitled Form", lang)
+    module = Module.new_module(_("Untitled Module"), lang)
```

Comment:
huh, I do not. Nor did I know that the underscore translates a string, so thanks for that.